meta_client/Cargo.toml

```diff
@@ -19,7 +19,7 @@ futures = { workspace = true }
 log = { workspace = true }
 prost = { workspace = true }
 rand = { workspace = true }
-reqwest = "0.11.13"
+reqwest = { version = "0.11.13", default-features = false, features = ["rustls-tls"] }
```

Comment:
I find that this dependency is never used. Maybe we can just remove it here.

Reply:
That's kind of awkward, I will file another issue to check unused deps.